src/components/blocks/InformationBlock/InformationBlock.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import InformationBlock from './InformationBlock';
+
+export default {
+  title: 'DataInColumn/DataInRow',
```

Comment:
Sorry, I think there was a misunderstanding
It should still be `Blocks/InformationBlock`

What should be changed is names below
NoDataRow - DataInColumn
WithDataRow - DataInRow

Because it looks like this in storybook
![image](https://user-images.githubusercontent.com/86728658/152370979-aefc519d-11cf-461b-a069-5fd6b4fc19e9.png)